packages/kit/src/views/Earn/components/showProtocolListDialog.tsx

```
@@ -236,17 +230,12 @@ function ProtocolListDialogContent({
-        }
-        renderItem={renderItem}
-        renderSectionHeader={renderSectionHeader}
-        SectionSeparatorComponent={<YStack h="$4" />}
-        stickySectionHeadersEnabled={false}
-        estimatedItemSize={62}
-      />
+      {protocolData.map((section) => (
+        <>
+          {renderSectionHeader({ section })}
+          {section.data.map((item) => renderItem({ item }))}
+        </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)